src/components/Field/Field.tsx

```diff
@@ -108,3 +116,7 @@ const Error = styled.p`
     line-height: 1.4;
   `}
 `
+
+const TagWrapper = styled.span`
+  margin-left: 8px;
```

Comment:
Use `theme.size.pxToRem(theme.size.space.xs)` Instead of specifying the value directly 🙏